apps/projects/app/index.js

```diff
@@ -1,5 +1,3 @@
-import '@babel/polyfill'
```

Comment:
I love seeing code get deleted, and even more so whole packages! But I'm not sure why this is disappearing now, in particular. Mind explaining what enabled us to get rid of this here?

It looks like we're already importing it in every app's `script.js` – is that why?

I also noticed that it's in every app's `package.json` _except_ for `address-book`. Should we add it there while we're improving this behavior?

Reply:
At first I had errors in console about... something irrelevant. I was told it is because '@babel/polyfill' is missing, and it should be there (because it is present in Aragon apps already, etc). I just retested that and found the errors are still there. Therefore this import should stay in place in all apps.